pkgs/servers/nosql/mongodb/mongodb.nix

```
@@ -44,7 +68,7 @@ in stdenv.mkDerivation rec {
     inherit sha256;
   };
 
-  nativeBuildInputs = [ variants.scons ];
+  nativeBuildInputs = [ variants.scons ] ++ lib.optionals (stdenv.isLinux && versionAtLeast version "4.4") [ lzma ];
```

```suggestion
  nativeBuildInputs = [ variants.scons ]
    ++ lib.optionals (stdenv.isLinux && versionAtLeast version "4.4") [ lzma ];
```
You are sure this is only needed on Linux?

---

Not sure for arm, but for x86_64-darwinthis is not needed.